### PR TITLE
Remove jobLocator from SimpleJobService

### PR DESCRIPTION
**Pull request by sylank**
resolves: #3405

### CODE REVIEW

**Comment by sylank** [comment]
I got Travis CI error: `The job exceeded the maximum log length, and has been terminated.`. And somehow I am not able to build it locally 😄 . But I am not able to build the master either. 
Could somebody take a look at this build? 😺

**Comment by cppwfs** [comment]
Hmm... The main build is not showing anything and I just built locally.  What are you seeing?

**Comment by sylank** [comment]
I have rebuilt it again and it is working except the `TaskCommandTests#testGetLog()`, that is always failing to me :D

File: spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/batch/SimpleJobService.java [hunk 1]
```diff
@@ -322,8 +240,7 @@ public int countJobExecutions() {
 
 	@Override
 	public int countJobs() {
-		Collection<String> names = new HashSet<String>(jobLocator.getJobNames());
-		names.addAll(jobInstanceDao.getJobNames());
+		Collection<String> names = new HashSet<>(jobInstanceDao.getJobNames());
```
Comment (markpollack) [inline spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/batch/SimpleJobService.java]: can use `return jobInstanceDao.getJobNames().size();`

**Comment by markpollack** [comment]
Resolved in a5cdfd980198b8f87a997994ec129d59b82da9eb

**Pull request closed without merging**